modules/mod_base/controllers/controller_postback.erl

```diff
@@ -28,6 +28,7 @@
 
 -include_lib("zotonic.hrl").
 
+-define(SIDEJOB_TIMEOUT, 1000).
```

Comment:
The comet connection is waiting a little bit before sending the first thing that arrives. Usually more arrives, and it is quite expensive to setup long-polling connections.

Is it possible to set longer timeouts for long tasks?